package/CHANGELOG

```diff
@@ -13,13 +13,23 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/?? IAlibay
+??/??/?? IAlibay, cbouy
 
  * 2.2.0
 
 Fixes
+  * When converting an AtomGroup to an RDKit molecule (PR #3044):
```

Comment:
Does this fix #3339 ? If so, the issue number should appear here. Even if it's only a partial fix — issue numbers are generally more important than PRs although we typically cite _both_ when possible.

Reply:
No, #3339 keeps track of the cases that this PR cannot fix